view/frontend/web/js/view/payment/method-renderer/adyen-google-pay-method.js

```diff
@@ -100,7 +100,7 @@ define(
                     }
                 });
                 var googlepay = self.checkoutComponent.create('paywithgoogle', {
-                    showPayButton: true,
+                    showPayButton: false,
```

Comment:
for Google pay we need to show the pay button - sorry that is an exception
The setting you are looking for is in the cc and hpp method js files.
Probably you will need to create the `showPayButton` configuration because it was by default false so we didn't need to do anything with it until now